samples/04_gis_analysts_data_scientists/how-much-green-is-Delhi-as-on-15-oct-2017.ipynb

```diff
@@ -86,7 +86,7 @@
    "metadata": {},
```

Comment:
<p>Can we update this comment to something like - 'Here we're getting the multispectral landsat imagery item', as we are changing search method to get</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>